multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -182,6 +182,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("MMScans", "https://mm-scans.org", "en", overrideVersionCode = 4),
         SingleLang("MadaraDex", "https://madaradx.org", "en", isNsfw = true),
         SingleLang("Manga Action", "https://mangaaction.com", "en", overrideVersionCode = 2),
+        SingleLang("Manga-Alarab.", "https://manga-alarab.com", "ar", overrideVersionCode = 1),
```

Comment:
You need to fix the extension name, it can't contain special characters directly.

```suggestion
        SingleLang("Manga Alarab", "https://manga-alarab.com", "ar"),
```